packages/react-core/single-packages.config.json

```diff
@@ -1,5 +1,5 @@
 {
   "packageName": "@patternfly/react-core",
   "moduleGlob": ["/dist/esm/helpers/**/*.js", "/dist/esm/styles/**/index.js", "/dist/esm/*/*/**/index.js"],
-  "exclude": ["/dist/esm/helpers/Popper/thirdparty"]
+  "exclude": ["/dist/esm/helpers/Popper/thirdparty", "/dist/esm/next"]
```

Comment:
Can't recall if we previously had to do anything else, but this resolves the build failing due to Next no longer exporting anything.